ci-operator/config/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-master.yaml

```diff
@@ -8,6 +8,14 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8
+  to: csi-driver-shared-resource-operator
```

Comment:
/cc @coreydaley 

Not sure how we got this far without proper image building/promotion. We are going to need this to deploy this operator via the cluster storage operator.